src/vscode-bicep/src/test/e2e/visualizer.test.ts

```diff
@@ -28,12 +29,15 @@ describe("visualizer", (): void => {
     // Give the language server sometime to finish compilation.
     await sleep(2000);
 
-    const viewColumn = await retryWhile(
-      async () => await executeShowVisualizerCommand(document.uri),
-      () => !visualizerIsReady(document.uri)
-    );
-
-    expect(visualizerIsReady(document.uri)).toBeTruthy();
+    const viewColumn = await executeShowVisualizerCommand(document.uri);
```

Comment:
The retry was there before because the command wouldn't run if commands weren't registered yet.  No longer needed and causes problems.  Just waiting until it's marked complete.